addons/website/static/src/xml/website.pageProperties.xml

```diff
@@ -156,6 +157,27 @@
                     </div>
                 </div>
             </div>
+            <div role="tabpanel" id="seo_page_info" class="tab-pane fade">
+                <div class="form-group row">
+                    <label class="col-form-label col-md-3" for="website_meta_title">Title</label>
+                    <div class="col-md-9">
+                        <input type="text" class="form-control form-control-sm" id="website_meta_title" t-att-value="widget.page.website_meta_title || ''" />
+                    </div>
+                </div>
+                <div class="form-group row">
+                    <label class="col-form-label col-md-3" for="website_meta_description">Description</label>
+                    <div class="col-md-9">
+                        <textarea class="form-control form-control-sm" id="website_meta_description" style="max-width: 100%;height:145px;max-height:145px"><t t-esc="widget.page.website_meta_description || ''"/></textarea>
+                        <div class="alert alert-warning mt16 small" id="website_meta_description_warning" style="display: none;"/>
+                    </div>
+                </div>
+                <div class="form-group row">
+                    <label class="col-form-label col-md-3" for="website_meta_keywords">Keywords</label>
```

Comment:
Can you add a tooltip like it is the case for Indexed and Share this page ?
eg: `<i class="fa fa-question-circle-o" title="Hide this page from search results" role="img" aria-label="Info"></i>`
And in tooltip something like "Separate keywords with a comma"

Reply:
Ok